drivers/analog/adc.c

```diff
@@ -420,6 +435,23 @@ static int adc_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
   return ret;
 }
 
+/****************************************************************************
+ * Name: adc_reset
+ ****************************************************************************/
+
+static int adc_reset(FAR struct adc_dev_s *dev)
+{
+  /* Set overrun flag to give read a chance to recover */
+
+  dev->ad_isovr = true;
+
+  /* No need to notify here. The adc_receive callback will be called next.
+   * If an adc overrun occurs then there must be at least one conversion.
```

Comment:
```suggestion
   * If an ADC overrun occurs then there must be at least one conversion.
```
nit: ADC acronym in all caps, for consistency.